up-console/up-console.csproj

```diff
@@ -2,15 +2,15 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
```

Comment:
@jennyf19 : at the same time I verified the PR, I've updated to .NET Core 3.0 ...

Reply:
cool. thanks @jmprieur